app/views/parties/members.html.haml

```diff
@@ -0,0 +1,6 @@
+%h1= t('.title')
+%ul.people-list
+  - @people.each do |person|
+    %li
+      = person.respond_to?(:personGivenName) ? person.personGivenName : ''
```

Comment:
this has been done. Please see new pull request for branch: aebirim/new-changes-party-controller